Phaethon/InternalApi/InternalApi.csproj

```diff
@@ -139,6 +139,10 @@
     <Compile Include="Migrations\201812021133435_Initial.Designer.cs">
       <DependentUpon>201812021133435_Initial.cs</DependentUpon>
     </Compile>
+    <Compile Include="Migrations\201812050952594_removed DispalyNames from job and customer tables.cs" />
```

Comment:
why do you keep adding migrations? we agreed that we do them only for master branch

Reply:
It's because of that db error with the DateTime. This is the workaround i found. (every time this error appears, delete all traces of db, including migrations, make new everything)

Reply:
well ok, than this is ok, but its just anoying that i have to delete the migration or make new just so my stuff would work

Reply:
We can remove this problem by switching away from EF 😄